### PR TITLE
Add %%COURSE_ID%% to supported keywords

### PR DESCRIPTION
**Pull request by kluo**
@caesar2164

### CODE REVIEW

File: lms/startup.py [hunk 1]
```diff
@@ -184,6 +184,10 @@ def course_display_name_sub(user, course):
         """ Returns the course's display name """
         return course.display_name
 
+    def course_id_sub(user, course):
```
Comment (stvstnfrd) [inline lms/startup.py]: Can we give this a better name? The current function name doesn't make its intent clear.

Also, the docstring text doesn't match the function logic.

Can you clarify?

File: common/djangoapps/util/keyword_substitution.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
         - %%USER_ID%% => anonymous user id
         - %%USER_FULLNAME%% => User's full name
         - %%COURSE_DISPLAY_NAME%% => display name of the course
+        - %%COURSE_ID%% => course URL
```
Comment (caesar2164) [inline common/djangoapps/util/keyword_substitution.py]: ```COURSE_ID``` isn't *EXACTLY* like "course URL"

Comment (caesar2164) [inline common/djangoapps/util/keyword_substitution.py]: ```- %%COURSE_ID%% => course identification path```

@stvstnfrd ?

File: lms/startup.py [hunk 3]
```diff
@@ -198,6 +202,7 @@ def course_end_date_sub(user, course):
         '%%USER_ID%%': Keyword(user_id_sub, 'anonymous_user_id (for use in survey links)'),
         '%%USER_FULLNAME%%': Keyword(user_fullname_sub, 'user profile name'),
         '%%COURSE_DISPLAY_NAME%%': Keyword(course_display_name_sub, 'display name of the course'),
+        '%%COURSE_ID%%': Keyword(get_course_id_string, 'course identifier'),
```
Comment (caesar2164) [inline lms/startup.py]: here your ```COURSE_ID``` string is different than it was in common/djangoapps/util/keyword_substitution.py

Comment (caesar2164) [inline lms/startup.py]: I actually like this one better than my suggestion

File: common/djangoapps/util/keyword_substitution.py (hunk 2 above)
Comment (caesar2164) [inline common/djangoapps/util/keyword_substitution.py]: ```- %%COURSE_ID%% => course identifier```

Comment (caesar2164) [inline common/djangoapps/util/keyword_substitution.py]: DOH, this is a docstring...

**Comment by caesar2164** [comment]
fix the doctoring, and I'm :+1:

File: lms/startup.py (hunk 1 above)
Comment (stvstnfrd) [inline lms/startup.py]: This looks so much better; thanks for the change!

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**